app/src/main/java/org/kinecosystem/kinit/viewmodel/SplashViewModel.kt

```diff
@@ -49,75 +39,20 @@ class SplashViewModel(var splashNavigator: SplashNavigator?) {
         analytics.logEvent(Events.Analytics.ViewSplashscreenPage())
     }
 
-    fun onRetryClicked(view: View?) {
-        analytics.logEvent(Events.Analytics.ClickRetryButtonOnErrorPage(Analytics.VIEW_ERROR_TYPE_ONBOARDING))
-        splashNavigator?.moveToSplashScreen()
-        showCreatingWallet.set(true)
-        checkReadyToMove()
-        servicesProvider.onBoardingService.appLaunch()
-    }
-
-    fun onContactSupportClicked(view: View?) {
-        analytics.logEvent(Events.Analytics.ClickContactLinkOnErrorPage(Analytics.VIEW_ERROR_TYPE_ONBOARDING))
-        splashNavigator?.openContactSupport()
-    }
-
-    fun onDestroy() {
-        splashNavigator = null
-        if (callback != null) {
-            walletReady.removeOnPropertyChangedCallback(callback)
-            callback = null
-        }
-    }
-
     private fun checkReadyToMove() {
         if (googlePlayServicesAvailable) {
-            if (walletReady.get()) {
-                moveToNextScreen()
-            } else {
-                userRepository.isFirstTimeUser = true
-                showCreatingWallet.set(true)
-                addWalletReadyCallback()
-                scheduleTimeout()
-            }
+            moveToNextScreen()
         }
     }
 
     private fun moveToNextScreen() {
-        if (userRepository.isFirstTimeUser
-            && userRepository.isPhoneVerificationEnabled
-            && !userRepository.isPhoneVerified) {
-            splashNavigator?.moveToTutorialScreen()
-        } else if (userRepository.isFirstTimeUser && !userRepository.isPhoneVerificationEnabled) {
+        if (userRepository.isFirstTimeUser || !userRepository.isPhoneVerified) {
```

Comment:
This should be 
if (!userRepository.isPhoneVerified)  
regardless if this is a first time user or not.